packages/lib/models/Setting.ts

```diff
@@ -2620,24 +2633,76 @@ class Setting extends BaseModel {
 		return '';
 	}
 
-	public static sectionNameToIcon(name: string) {
-		if (name === 'general') return 'icon-general';
-		if (name === 'sync') return 'icon-sync';
-		if (name === 'appearance') return 'icon-appearance';
-		if (name === 'note') return 'icon-note';
-		if (name === 'folder') return 'icon-notebooks';
-		if (name === 'plugins') return 'icon-plugins';
-		if (name === 'markdownPlugins') return 'fab fa-markdown';
-		if (name === 'application') return 'icon-application';
-		if (name === 'revisionService') return 'icon-note-history';
-		if (name === 'encryption') return 'icon-encryption';
-		if (name === 'server') return 'far fa-hand-scissors';
-		if (name === 'keymap') return 'fa fa-keyboard';
-		if (name === 'joplinCloud') return 'fa fa-cloud';
+	public static sectionMetadataToSummary(metadata: SettingMetadataSection): string {
+		// TODO: This is currently specific to the mobile app
+		const sectionNameToSummary: Record<string, string> = {
+			'general': _('Language, date format'),
+			'appearance': _('App theme, editor font'),
+			'sync': _('Sync, encryption, proxy'),
+			'joplinCloud': _('Joplin Cloud settings'),
+			'markdownPlugins': _('Markdown plugins: Media player, math, footnotes, soft breaks, ...'),
```

Comment:
Technically those are plugins, but from a user perspective they are just options to turn on or off, so we don't need to mention "Markdown plugins". Maybe instead: "Media player, math, diagrams, table of content" (No "...")